### DISCUSSION TITLE
Trying to setup an outdoor enclosure, privacy, Pixel phone, & Birdnet-Pi vs Go

### DISCUSSION THREAD

**Discussion by Kavega** [tphakala/birdnet-go#321] [General]
Hi all, 

I have played around a bit with Birdnet-GO on my computer and am really impressed. I am really looking forward to setting up an outdoor enclosure with a rasberry pi most likely and running this to autodetect birds in my area. However my region is very strict with privacy and I wanted to ensure no human voices are ever recorded. Ideally I just need the species list without ever having to connect to the internet as we don't have connection where I want to set it up. 

1. I am very new to this and while I feel that BirdNet-Go is the perfect solution but I don't really understand the different between Birdnet-Pi and Go. Both can be entirely on device and just ID the birds no? Or is Go the full model on device and Pi needs some sort of connection? 

2. I do have an extra pixel phone, I was wondering if it is even in the realm of possibility to get Go to run on it rather than need to get and install a rasberry pi. 

Thank you very much for any help you can offer!

**Comment by tphakala** [comment]
BirdNET-Pi and BirdNET-Go are two different applications from separate development groups for essentially the same use case. Both are based on the same BirdNET Analyzer AI model (https://github.com/kahst/BirdNET-Analyzer).

BirdNET-Pi:
* Has been in development since around 2020; however, the original developer abandoned the project a few years ago. There are some forks with active development
* Is based on Python language and is essentially a set of scripts
* Runs best on Linux systems (may run on Windows but this isn't confirmed)
* Records audio to SD card (typically 15 to 30 seconds) and then analyzes these audio clips to produce detections, which significantly reduces the life expectancy of SD cards
* Has a comprehensive web user interface

BirdNET-Go:
* Development started in October 2023, primarily as a learning project for Go programming language and to improve upon BirdNET-Pi's foundation
* Is programmed in Go language and compiles into a relatively high-performance binary application
* Runs on Linux, macOS, and Windows
* Captures audio directly to RAM for analysis, resulting in minimal wear on SD cards for Raspberry Pi deployments
* Is still early in development, with the web user interface lacking several planned features

Both solutions analyze audio locally with no need for internet connectivity. While both support BirdWeather (a third-party service for publishing detections on a map), this feature is optional.

Regarding phone-based deployments: These are not supported by BirdNET-Go; a Raspberry Pi is recommended.

**Reply by HWittingen** [reply]
There is one more major difference between the two. Which is about accessibility for the visually impaired people.
BirdNET-Pi webUI is rather poorly or not accessible. Specific the daily summary is generated as an image.
BirdNET-Go webUI still has some limitations. But after some guidance a blind friend was able to interpret the daily summary using an screen reader. And he also was able to find some specific owl detection and play and download the related recording.
